content/chat.adoc

```diff
@@ -3,6 +3,30 @@ layout: simplepage
 title: Chat
 ---
 
+= Gitter
+
+The Jenkins community discusses various topics in multiple rooms on https://gitter.im/jenkinsci/home[Gitter] 
+
+== Gitter Rooms
+
+=== https://gitter.im/jenkinsci/jenkins[Jenkins]
+
+General discussion about the Jenkins Continuous Integration Server
```

Comment:
We have not called Jenkins "Continuous Integration" for years.

Reply:
I see now this is probably taken from the room description. I changed it to "Jenkins automation server" matching the https://github.com/jenkinsci/jenkins/ repo description.